### ISSUE TITLE
Home directory creation

### ISSUE THREAD

**Issue by nicholasmello** [xonmello/arcade-builder#25]
Home directory is not being created now that home is a separate partition

**Comment by nicholasmello** [comment]
Temp fix ready after unblock from #26

**Comment by nicholasmello** [comment]
```
# ls /home/
arcade
# ls /home/arcade/
# ls /etc/init.d/
S01syslogd      S02klogd        S02sysctl       S20seedrng      S40network      S50dropbear     S80dhcp-relay   S80dhcp-server  rcK             rcS
```
Fixed the there is no script needed... looks like my weirdness I was initially doing with the skeleton before I got a more permanent solution, combined with the move to a separate file system, might have caused the bug. Closing for now.
